source/documentation/managing_apps/autoscaling.erb

```diff
@@ -0,0 +1,282 @@
+## Autoscaling
+
+Apps can be configured to dynamically scale in and out based on autoscaling
+policies.
+
+Scaling your app to fewer instances can reduce the cost during periods of low
+usage,
+eg at night.
```

Comment:
Unnecessary newline

Reply:
Can I resolve the "unnecessary newline" comments as they don't appear in the rendered markdown, or is this a comment about markdown style?

Reply:
Happy to have them resolved. I assumed they appeared in the rendered HTML too.

Reply:
You need two consecutive newlines to achieve a line break